mongo/src/main/java/org/axonframework/mongo/eventsourcing/eventstore/MongoTrackingToken.java

```diff
@@ -37,7 +38,7 @@
  *
  * @author Rene de Waele
  */
-public class MongoTrackingToken implements TrackingToken {
+public class MongoTrackingToken implements TrackingToken, Serializable {
```

Comment:
@abuijze also pointed out we'd like to have the required Jackson annotations on the `MongoTrackingToken` to allow JacksonSerializers to work with this object too.